ignore/src/types.rs

```diff
@@ -107,7 +108,7 @@ const DEFAULT_TYPES: &'static [(&'static str, &'static [&'static str])] = &[
     ("bazel", &["*.bzl", "WORKSPACE", "BUILD"]),
     ("bitbake", &["*.bb", "*.bbappend", "*.bbclass", "*.conf", "*.inc"]),
     ("bzip2", &["*.bz2"]),
-    ("c", &["*.c", "*.h", "*.H"]),
+    ("c", &["*.c", "*.h", "*.H", "*.cats"]),
```

Comment:
I don't think I've seen `cats` used for a C file before. Where does that come from?

Reply:
It is used with ATS to contain C code called by ATS code later, e.g. https://groups.google.com/forum/#!topic/ats-lang-users/On6L-1e6p_0

Reply:
Great. Thank you.